pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -134,7 +135,6 @@ in rec {
 
     patches = [
       ./0000-fix-SPDK-build-env.patch
-      ./ceph-glibc-2-32-sigdescr_np.patch
```

Comment:
Is that patch unused? If so it should be deleted.

Reply:
Ah yes, brainfart, thanks :)